tests/synthesizer/test_eleven_labs.py

```diff
@@ -0,0 +1,49 @@
+from pydantic import ValidationError
+import pytest
+from vocode.streaming.synthesizer.base_synthesizer import SynthesisResult
+from vocode.streaming.synthesizer.eleven_labs_synthesizer import ElevenLabsSynthesizer
+from vocode.streaming.models.synthesizer import ElevenLabsSynthesizerConfig
+from aioresponses import aioresponses
+from vocode.streaming.models.message import BaseMessage
+from vocode.streaming.models.audio_encoding import AudioEncoding
+
+@pytest.mark.asyncio
+async def test_with_api_key(eleven_labs_synthesizer_with_api_key: ElevenLabsSynthesizer, mock_eleven_labs_api: aioresponses):
+    response = await eleven_labs_synthesizer_with_api_key.create_speech(BaseMessage(text="Hello, world!"), 1024)
+    assert(isinstance(response, SynthesisResult))
+    assert(response.chunk_generator is not None)
+    
+@pytest.mark.asyncio
+async def test_with_wrong_api_key(eleven_labs_synthesizer_wrong_api_key: ElevenLabsSynthesizer, mock_eleven_labs_api: aioresponses):
+    with pytest.raises(Exception, match="ElevenLabs API returned 401 status code"):
+        await eleven_labs_synthesizer_wrong_api_key.create_speech(BaseMessage(text="Hello, world!"), 1024)
+        
+@pytest.mark.asyncio
+async def test_with_env_api_key(eleven_labs_synthesizer_env_api_key: ElevenLabsSynthesizer, mock_eleven_labs_api: aioresponses):
+    response = await eleven_labs_synthesizer_env_api_key.create_speech(BaseMessage(text="Hello, world!"), 1024)
+    assert(isinstance(response, SynthesisResult))
```

Comment:
separate these two lines into a helper method called `assert_synthesis_result_valid`

Reply:
this method should also read the audio chunks from the generator into an `AudioSegment` - but not assert anything about that `AudioSegment`